cdbot/cogs/fun.py

```diff
@@ -253,7 +260,11 @@ def check(reaction, user):
         Returns a random quotation from the #quotes channel.
         A user can be specified to return a random quotation from that user.
         """
-        conn = await asyncpg.connect()
+        conn = await asyncpg.connect(host=base64.b64decode(environ.get("PGHOST")),
+                                     port=base64.b64decode(environ.get("PGPORT")),
+                                     user=base64.b64decode(environ.get("PGUSER")),
```

Comment:
This is not nice, could this be stored in the constants file so you can pass it straight into the function?